Sources/KafkaTestUtils/KafkaTestMessages.swift

```diff
@@ -0,0 +1,62 @@
+import Kafka
+import struct Foundation.Date
+import Logging
+
+public struct KafkaTestMessages {
+    public static func sendAndAcknowledge(
+        producer: KafkaProducer,
+        events: KafkaProducerEvents,
+        messages: [KafkaProducerMessage<String, String>],
+        logger: Logger = .kafkaTest
+    ) async throws {
+        for message in messages {
+            while true { // Note: this is an example of queue full
+                do {
+                    try producer.send(message)
+                    break
+                } catch let error as KafkaError where error.description.contains("Queue full") {
+                    continue
+                } catch {
+                    logger.error("Caught some error: \(error)")
+                    throw error
+                }
+            }
+        }
+        
+        var receivedDeliveryReportsCtr = 0
```

Comment:
```suggestion
        var receivedDeliveryReportsCounter = 0
```

Reply:
Generally, we try to avoid abbreviations unless they make sense (e.g. `HTTP`), please also do that for the other abbreviations you used